tests/test_general.py

```diff
@@ -356,6 +357,86 @@ def test_user_marker_points(self):
         self.assertEqual(105, elementOut.getIdentifier())
         assertAlmostEqualList(self, [0.3452673123795837, 1.0, 0.6634646029995092], xiOut, delta=TOL)
 
+    def test_deletion(self):
+        """
+        Test deletion of element ranges on a stomach scaffold with scaffold package.
+        """
+        scaffoldPackage = ScaffoldPackage(MeshType_3d_stomach1)
```

Comment:
Future comment, no changes needed now.
The stomach is a large slow scaffold which slows down the otherwise pretty fast general tests.
At some point we should replace it with a box scaffold with 2 elements and hack in auto/built-in annotation groups, plus add user annotation groups to test.